examples/frontend/containers/Layout.tsx

```diff
@@ -54,11 +60,15 @@ export default function Layout({ children, project = 'share' }) {
 
     const preSignOut = () => {
         localStorage.removeItem('contractAddress')
-        signOut()
         router.reload(window.location.origin + router.asPath)
+        signOut()
```

Comment:
Will this do something even if the router gets reloaded before?

Reply:
Actually nope. After hours of testing, it gets swapped. :-)